package.json

```diff
@@ -12,8 +12,8 @@
   "workspaces": [
     "packages/bridge",
     "packages/braze",
-    "packages/react-18",
-    "packages/react-dom-18",
+    "packages/react",
+    "packages/react-dom",
```

Comment:
This will mean that we won't be updating react 18 in the future. I guess thats what we want right? We discussed at some point that we would like users stay update with the latest yea?

Reply:
Yeah, I don't imagine it'll get more updates, but we can keep it around of course.